### PR TITLE
🧮 Videos over-fetching

### PR DESCRIPTION
**Pull request by WRadoslaw**
#4322

A new way of suggestion: 
1. Fetch a newer video than the current one 
2. If there is no newer video, fetch the oldest

### CODE REVIEW

File: packages/atlas/src/components/_video/VideoPlayer/VideoOverlay.tsx [hunk 1]
```diff
@@ -31,28 +34,41 @@ export const VideoOverlay: FC<VideoOverlayProps> = ({
...
+  const { videos, refetch } = useBasicVideos(
     {
+      limit: 1,
+      orderBy: VideoOrderByInput.ChannelCreatedAtAsc,
       where: {
+        ...publicVideoFilter,
         channel: {
           id_eq: channelId,
         },
+        createdAt_gt: currentVideoTimestamp,
```
Comment (drillprop) [inline packages/atlas/src/components/_video/VideoPlayer/VideoOverlay.tsx]: The name `currentVideoTimestamp` is really misleading. At first glance, I thought you mean the current time of the actual video(in the video player). Can we rename this to something more intuitive - maybe even `currentVideoCreatedAt`?

**Review by drillprop — CHANGES_REQUESTED** [review]

File: packages/atlas/src/components/_video/VideoPlayer/VideoOverlay.tsx (hunk 1 above)
Comment (attemka) [reply packages/atlas/src/components/_video/VideoPlayer/VideoOverlay.tsx]: agree, should be easier to match to `video.createdAt` prop this way

File: packages/atlas/src/components/_video/VideoPlayer/VideoOverlay.tsx [hunk 2]
```diff
@@ -31,28 +34,41 @@ export const VideoOverlay: FC<VideoOverlayProps> = ({
...
-    if (!videos?.length || videos.length <= 1) {
+    if (!videos?.length || videos.length === 0) {
+      refetch({
+        where: {
+          channel: {
+            id_eq: channelId,
+          },
+          createdAt_lt: currentVideoCreatedAt,
+        },
+      })
```
Comment (drillprop) [inline packages/atlas/src/components/_video/VideoPlayer/VideoOverlay.tsx]: Sorry, I missed that before. I tested this on video /video/552 and I think this is not working correctly. Channel has two videos and it's not switching to the other one.

**Review by drillprop — CHANGES_REQUESTED** [review]

**Review by attemka — APPROVED** [review]

**Review by drillprop — APPROVED** [review]
LGTM as well

**Pull request merged**